sql/src/main/java/io/crate/planner/statement/CopyStatementPlanner.java

```diff
@@ -245,12 +286,12 @@ private static void rewriteToCollectToUsePartitionValues(List<Reference> partiti
 
     /**
      * To generate the upsert request the following is required:
-     *
-     *  - relationName + partitionIdent / partitionValues
-     *      -> to retrieve the indexName
-     *
-     *  - primaryKeys + clusteredBy  (+ indexName)
-     *      -> to calculate the shardId
+     * <p>
+     * - relationName + partitionIdent / partitionValues
+     * -> to retrieve the indexName
+     * <p>
+     * - primaryKeys + clusteredBy  (+ indexName)
+     * -> to calculate the shardId
```

Comment:
Is this auto-format? Seems ok like it was.

Reply:
will revert the auto-format of the file, was done by mistake